cfme/tests/containers/test_ssl_providers.py

```diff
@@ -0,0 +1,121 @@
+        try:
+            fill(provider.credentials[cred].form, provider.credentials[cred], validate=True, validate_hawkular=True)
+            provider._submit(False, provider.add_provider_button)
+            flash.assert_message_contain('Containers Providers "' + provider_name + '" was saved')
+        except:
+            # The refresh is a workaround for
+            # https://github.com/ManageIQ/integration_tests/issues/4478
+            sel.refresh()
+            soft_assert(False, provider_name + ' wasn\'t added successfully using ' +
+                        default_sec_protocols + ' and ' +
+                        hawkular_sec_protocols + ' hawkular security protocol')
+
+
+def form_data(provider, default_sec_protocol, hawkular_sec_protocol=None, with_hawkular=False):
+    name_text = conf.cfme_data.get('management_systems', {})[provider.key].get('name', [])
+    type_select = 'OpenShift Container Platform'
+    hostname_text = conf.cfme_data.get('management_systems', {})[provider.key].get('hostname', [])
+    port_text = conf.cfme_data.get('management_systems', {})[provider.key].get('port', [])
+    zone_select = 'default'
+    default_ca_certificate = get_certificate(provider, sec_protocol=default_sec_protocol)
+    hawkular_ca_certificate = get_certificate(provider, sec_protocol=hawkular_sec_protocol)
+    if with_hawkular:
+        return {'name_text': name_text,
+                'type_select': type_select,
+                'hostname_text': hostname_text,
+                'port_text': port_text,
+                'trusted_ca_certificates': default_ca_certificate,
+                'sec_protocol': default_sec_protocol,
+                'zone_select': zone_select,
+                'hawkular_hostname': conf.cfme_data.get('management_systems',
+                                                        {})[provider.key].get('hostname', []),
+                'hawkular_api_port': '443',
+                'hawkular_sec_protocol': hawkular_sec_protocol,
+                'hawkular_ca_certificates': hawkular_ca_certificate,
+                }
+    else:
+        return {'name_text': name_text,
+                'type_select': type_select,
+                'hostname_text': hostname_text,
+                'port_text': port_text,
+                'trusted_ca_certificates': default_ca_certificate,
+                'sec_protocol': default_sec_protocol,
+                'zone_select': zone_select,
+                }
+
+
+def get_certificate(provider, sec_protocol):
+    if sec_protocol != 'SSL trusting custom CA':
+        return None
+    else:
+        ip = conf.cfme_data.get('management_systems', {})[provider.key].get('hostname', [])
+        ssh_kwargs = {
+            'username': get_crud('cm-env1').credentials['token'].principal,
+            'password': get_crud('cm-env1').credentials['token'].secret,
+            'hostname': ip
+        }
+        ssh = SSHClient(**ssh_kwargs)
+        rc, cert = ssh.run_command('cat /etc/origin/master/ca.crt')
```

Comment:
You can do it like this:
```python
cert = ssh.run_command('cat /etc/origin/master/ca.crt')
assert cert
return str(cert)
```

The ssh result namedtuple has some methods that let it act like a bool ($?==0), int ($?) or string (the output)